astropy/units/equivalencies.py

```diff
@@ -508,6 +508,59 @@ def convert_K_to_Jy(x_K):
     return [(astrophys.Jy, si.K, convert_Jy_to_K, convert_K_to_Jy)]
 
 
+def thermodynamic_temperature(disp, T_cmb=2.7255*si.K):
+    r"""Defines the conversion between Jy/beam and "thermodynamic temperature",
+    :math:`T_{CMB}`, in Kelvins.  The thermodynamic temperature is a unit very
+    commonly used in cosmology.  See, e.g., "Planck 2013 results. IX. HFI spectral response"
+    (Planck Collaboration 2013) eqn 8 (available on `arXiv <https://arxiv.org/abs/1303.5070>`__).
+
+    :math:`T_{CMB} \equiv B_\nu / \left(2 k \nu^2 / c^2  f(\nu) \right)`
```

Comment:
I'm actually rather confused about what temperature the equivalency gives. The parameter names suggest `t_cmb` is passed in, but from the code below, I think `t_cmb` is only used for calculating `f(\nu)`. But then this equation suggests `T_CMB` is in fact the result. Isn't this meant to be a temperature offset (which should be small)?